partcad-cli/src/partcad_cli/click/commands/inspect.py

```
@@ -0,0 +1,109 @@
+    "-p",
+    "--param",
+    "params",
+    metavar="<param_name>=<param_value>",
+    help="Assign a value to the parameter",
+)
+@click.argument("object", type=str, required=False)  # help="Part (default), assembly or scene to test"
+@click.pass_context
+@click.pass_obj
+def cli(ctx, context, verbal, package, interface, assembly, sketch, params, object):
+    params = {}
+    if not params is None:
```

Ruff (0.8.0)

Test for object identity should be `is not`

Convert to `is not`

(E714)